### PR TITLE
Fixed Java-8-related build problem. Ignored a failing test.

### PR DESCRIPTION
**Pull request by dpisarenko**
Issue: https://github.com/jcabi/jcabi-aspects/issues/124

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks, I'll ask someone to do a code review here, thanks for your pull request

**Comment by dmarkov** [comment]
@carlosmiranda could you please review this pull request

**Comment by carlosmiranda** [comment]
@dpisarenko Thanks, pretty small branch, I don't have comments.

**Comment by carlosmiranda** [comment]
@rultor Mind if we merge now?

**Comment by yegor256** [comment]
@rultor try to merge (I'm sure you will fail)

**Comment by dpisarenko** [comment]
@yegor256 

> I'm sure you will fail

Why?

File: src/test/java/com/jcabi/aspects/CacheableTest.java [hunk 1]
```diff
@@ -53,8 +54,11 @@
     /**
      * Cacheable can cache calls.
      * @throws Exception If something goes wrong
+     * @todo Make sure that this test runs under Java 8 and remove the Ignore
```
Comment (carlosmiranda) [inline src/test/java/com/jcabi/aspects/CacheableTest.java]: Puzzle format here is wrong. Should be `@todo #<current issue number>:<estimate>`. E.g. `@todo #126:30min`.

**Comment by carlosmiranda** [comment]
@yegor256 @dpisarenko Yes it will fail, sorry I missed something. One comment above.

**Comment by dpisarenko** [comment]
@carlosmiranda I fixed the `todo` statement.

**Comment by dpisarenko** [comment]
@yegor256 I think there is a problem with rultor. See https://github.com/yegor256/rultor/issues/738 .

**Comment by dpisarenko** [comment]
@yegor256 Accoring to http://www.rultor.com/t/1605-70236821 the PR was merged, but it still has the status open (not merged). Can you please look at it?

**Comment by carlosmiranda** [comment]
@rultor Let's try again to merge please.

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by dmarkov** [comment]
@carlosmiranda paid, thanks, added **19 mins** to your account, payment `50597732`

added +19 to your rating, now it is equal to [+2012](http://www.netbout.com/b/35023?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**